pkg/volume/csi/csi_attacher_test.go

```diff
@@ -104,67 +104,62 @@ func markVolumeAttached(t *testing.T, client clientset.Interface, watch *watch.R
 }
 
 func TestAttacherAttach(t *testing.T) {
-
+	defer utilfeaturetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CSIInlineVolume, true)()
```

Comment:
Can you structure these tests into 4 sections?
* Existing tests with the feature off
* New tests with the feature off: this validates that we fail appropriately when the feature is disabled
* Existing tests with the feature on: this validates that the new feature has not broken the existing behavior
* New tests with the feature on: this validates the new functionality

Reply:
Ok will give that a try.